### ISSUE TITLE
help conflict : need fixing help tag; that change to plugin-prefix using

### ISSUE THREAD

**Issue by tsuyoshicho** [tweekmonster/braceless.vim#52]
## in help-writing

```
TAGS

To define a help tag, place the name between asterisks (*tag-name*).  The
tag-name should be different from all the Vim help tag names and ideally
should begin with the name of the Vim plugin.  The tag name is usually right
aligned on a line.
```

## for example
`[j` in [braceless\.vim](https://github.com/tweekmonster/braceless.vim) and [vim-anyfold](https://github.com/pseewald/vim-anyfold) and helptag exception occur:

```
Vim(helptags):E154
```

rel: pseewald/vim-anyfold#27

**Comment by blueyed** [comment]
What is the conflicting tag name?

**Comment by tsuyoshicho** [comment]
I see

Try it

1. install  braceless.vim and vim-anyfold.
2. `:helptag` generate together.
3. occur above error.

and then;

4. `help [j` open one side (half user miss hit)

**Comment by pseewald** [comment]
I think it can not be avoided that different vim plugins define the same tags. Indeed the `helptags` command works on a specific directory and the documentation of each plugin is in its own `doc` directory so there should never be any conflict. The only way I could reproduce above error is to copy the documentation file of both plugins into the same directory and then run `:helptags` on that directory. With a plugin manager (vundle or pathogen), helptag generation works fine.

**Comment by tsuyoshicho** [comment]
Yes, but any plugin manger some plugins gater one because keep petformance.(rtp at small keep)

I can be workaround(set split/standalone picked plugin). but it is no roor cause fix.
Suggest this issue; to under official help guideline.

If reject this, I understood policy and OK.
